python/servo/build_commands.py

```diff
@@ -82,7 +82,7 @@ def notify_darwin(title, text):
         centre = Foundation.NSUserNotificationCenter.defaultUserNotificationCenter()
         centre.scheduleNotification_(note)
     except ImportError:
-        raise Exception("Please make sure that the Python pyobjc module is installed!")
+        raise Exception("Optional python module 'pyobjc' is not installed.")
```

Comment:
Can you make the same changes to the Linux message [here](https://github.com/tareqak/servo/blob/350bd4640dbeeccd1157602ed7dba1371492d11c/python/servo/build_commands.py#L40)? Also, can you capitalize the 'P' in Python? Thanks!